tslint.json

```diff
@@ -6,6 +6,7 @@
     ],
     "rules": {
       "interface-name": [true, "never-prefix"],
-      "prefer-object-spread": false
+      "prefer-object-spread": false,
+      "no-implicit-dependencies": false
```

Comment:
I'd like to figure out how to avoid changing this.

Reply:
OK. I'll look into it. By the way, I referred to the following page in order to fix this problem.

http://budiirawan.com/tslint-module-is-not-listed-as-dependency-in-package-json-no-implicit-dependencies/

Reply:
I wanted to do without changing tslint.json, but I could not find a solution. So I added the dev option after restoring the tslint.json. https://github.com/probot/probot/pull/625/commits/21bcb6026856111761a59154c6219d70eb083ffe

[Rule: no-implicit-dependencies](https://palantir.github.io/tslint/rules/no-implicit-dependencies/)

> By default the rule looks at "dependencies" and "peerDependencies". By adding the "dev" option the rule also looks at "devDependencies".